app/services/solidus_square/presenters/base_builder.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module SolidusSquare
+  module Presenters
+    class BaseBuilder
```

Comment:
I love this change, but considering this is a presenter do you mind moving it under `app/presenters/solidus_square/base_presenter.rb` and following the design pattern for the presenter?
You can find an example [here](https://www.rubyguides.com/2019/09/rails-patterns-presenter-service/)

Reply:
https://github.com/nebulab/solidus_square/pull/43/commits/f05c5bdb9c1fc91c8fbcf707118a24a9c77a255e